src/kyc_endpoint.py

```diff
@@ -2,14 +2,15 @@
 
 from flask import Blueprint, request, render_template, flash, redirect, make_response, abort
 
-from app_core import app, db, limiter
+from app_core import app, db, limiter, csrf
 from models import KycRequest
 import websocket
 import kyc_core
 
 logger = logging.getLogger(__name__)
 kyc = Blueprint('kyc', __name__, template_folder='templates')
 limiter.limit("100/minute")(kyc)
+csrf.exempt(kyc)
```

Comment:
I think we just want to exempt the one `/aplyid_webhook` route

Reply:
apply the change.